ext/DataFramesExt.jl

```diff
@@ -0,0 +1,47 @@
+module DataFramesExt
+
+using NEOs
+using NEOs: AbstractAstrometry
+
+import NEOs: RadecMPC, RadarJPL
+
+if isdefined(Base, :get_extension)
+    using DataFrames
+    import DataFrames: DataFrame
+else
+    using ..DataFrames
+    import ..DataFrames: DataFrame
+end
+
+# Method to convert a vector of observations to DataFrame
+function DataFrame(obs::Vector{T}) where {T <: AbstractAstrometry}
```

Comment:
While this should work fine for most purposes, I wonder how it compares in terms of performance and memory use to other solutions via Tables.jl overloadings (e.g. [here](https://discourse.julialang.org/t/mapping-vector-mytype-to-a-dataframe/24797/13)) or via StructArrays.jl (e.g. [here](https://discourse.julialang.org/t/mapping-vector-mytype-to-a-dataframe/24797/14))? Doing some quick benchmarks could really help us to select between alternatives

Reply:
(If indeed they apply, which they might not..)

Reply:
I did the following benchmarks by converting `data/99942.dat` to `DataFrame`:
- It turns out that `DataFrames` already has a method that allows to convert a vector of struct to `DataFrame` and it performs as follows:
```julia
298.700 μs (323 allocations: 1.15 MiB)
```
- The method proposed by this PR is not quite efficient:
```julia
26.391 ms (384091 allocations: 25.54 MiB)
```
- By overloading `Tables.jl` we get and amazing performance:
```julia
193.000 μs (76 allocations: 1.14 MiB)
```
Althouh this would imply that the weakdep is `Tables.jl` instead of `DataFrames.jl`.
- By using `StructArrays.jl` we get a slightly worse performance:
```julia
324.000 μs (104 allocations: 2.27 MiB)
```

It is clear that overloading `Tables.jl` is the most efficient option but it requires changing the weakdep. @PerezHz , @lbenet , should I proceed with this change?

Reply:
> It is clear that overloading Tables.jl is the most efficient option but it requires changing the weakdep. @PerezHz , @lbenet , should I proceed with this change?

Thanks for looking into this! I would say yes, let's go for it!